app/controllers/organizations/default_pet_tasks_controller.rb

```diff
@@ -1,7 +1,54 @@
 class Organizations::DefaultPetTasksController < Organizations::BaseController
   layout "dashboard"
 
+  before_action :verify_organization_for_current_user, only: [:new, :create, :edit, :update, :destroy]
+
   def index
     @default_pet_tasks = DefaultPetTask.all
   end
+
+  def new
+    @task = DefaultPetTask.new
+  end
+
+  def create
+    @task = DefaultPetTask.new(task_params)
+
+    if @task.save
+      redirect_to default_pet_tasks_path, notice: "Default pet task saved successfully."
+    else
+      flash.now[:alert] = "Error creating default pet task."
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @task = DefaultPetTask.find_by(id: params[:id])
+    raise ActiveRecord::RecordNotFound if @task.nil?
```

Comment:
`.find` will already raise `ActiveRecord::RecordNotFound` exception if the record is missing so we can shorten this to:
```suggestion
    @task = DefaultPetTask.find(params[:id])
```